doc/VERSIONING.md

```diff
@@ -10,3 +12,18 @@ a feature or a bugfix.
 Accordingly, if labels named `semver-major` or `semver-minor` are added to any
 issue or pull request before we have released `v1.0.0`, they actually assume
 minor and patch subversions.
+
+## Protocol versioning
+
+Before the package reaches version 1.0.0, the protocol version is bound to the
+first version of the `metarhia-jstp` package that implements the specific
+specification of the protocol. It would be nice to have it versioned
```

Comment:
@belochub reworded the sentence for clarity and to be more stylistically correct.